pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -65,6 +64,9 @@ type BindMechanism interface {
 	// binding and can be used in phase2 only.
 	decorateConfig() error
 	startDHCP(vmi *v1.VirtualMachineInstance) error
+
+	createVirtualNetworkingDevice(deviceName string, isMultiqueue bool, launcherPID int) error
+	configureVirtualNetworkingDevice() error
```

Comment:
These two are called from a single place one right after another. I don't think it merits two separate methods in an interface. Could we merge them on the API level?

Reply:
+1
maybe just call it createAndBindTapToBridge ? :)

Reply:
Can do; done.